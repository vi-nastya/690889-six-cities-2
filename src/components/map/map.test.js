import React from "react";
import renderer from "react-test-renderer";
import Map from "./map";

jest.mock(`leaflet`, () => ({
  icon: jest.fn(),
  map: jest.fn().mockReturnValue({
    setView: jest.fn(),
    remove: jest.fn()
  }),
  tileLayer: jest.fn().mockReturnValue({
    addTo: jest.fn()
  }),
  marker: jest.fn().mockReturnValue({
    addTo: jest.fn()
  }),
}));

it(`Map component is rendered correctly after relaunch`, () => {
  const tree = renderer.create(<Map points={[]}/>).toJSON();

  expect(tree).toMatchSnapshot();
});
