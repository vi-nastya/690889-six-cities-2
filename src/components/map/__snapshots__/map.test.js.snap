// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Map component is rendered correctly after relaunch 1`] = `
<div
  id="map"
/>
`;
